### PR TITLE
Fix: Import images

### PR DESCRIPTION
**Pull request by TheHienDuong**
## OVERVIEW
- Import src
- Update data

### CODE REVIEW

File: practice-one/src/Components/Section/Footer.jsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import footerIcon from '../../database/footerIcon';
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Footer.jsx]: MIssing comment

File: practice-one/src/Components/Section/Footer.jsx [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import footerIcon from '../../database/footerIcon';
+import logo from '../../assets/logo.png';
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Footer.jsx:8]: missing comment

File: practice-one/src/Components/Section/Footer.jsx (hunk 1 above)
Comment (phattruong-agilityio) [reply practice-one/src/Components/Section/Footer.jsx]: icons are in database?

File: practice-one/src/Components/Section/Navbar.jsx [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import logo from '../../assets/logo.png';
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Navbar.jsx:5]: missing comment

File: practice-one/src/database/footerIcon.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
+import Twitter__icon from '../assets/Twitter.png';
```
Comment (phattruong-agilityio) [inline practice-one/src/database/footerIcon.js:2]: missing comment

File: practice-one/src/database/productCard.js [hunk 5]
```diff
@@ -0,0 +1,130 @@
+import service__img from '../assets/service__img.png';
```
Comment (phattruong-agilityio) [inline practice-one/src/database/productCard.js:2]: Missing comment

File: practice-one/src/Components/Section/Footer.jsx (hunk 1 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Footer.jsx]: Done, added comment for this line

File: practice-one/src/Components/Section/Footer.jsx (hunk 2 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Footer.jsx:8]: Done, added comment for this line

File: practice-one/src/Components/Section/Navbar.jsx (hunk 3 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Navbar.jsx:5]: Done, added comment for this line

File: practice-one/src/database/footerIcon.js (hunk 4 above)
Comment (TheHienDuong) [reply practice-one/src/database/footerIcon.js:2]: Done, added comment for this line

File: practice-one/src/database/productCard.js (hunk 5 above)
Comment (TheHienDuong) [reply practice-one/src/database/productCard.js:2]: Done, added comment for this line

File: practice-one/src/Components/Section/Footer.jsx [hunk 6]
```diff
@@ -1,6 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+// Import footerIcon data from the database
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Footer.jsx]: Please simple comment. "Mock database"

File: practice-one/src/Components/Section/Footer.jsx (hunk 2 above)
Comment (phattruong-agilityio) [reply practice-one/src/Components/Section/Footer.jsx:8]: Please simple comment. "Images"

File: practice-one/src/Components/Section/Navbar.jsx (hunk 3 above)
Comment (phattruong-agilityio) [reply practice-one/src/Components/Section/Navbar.jsx:5]: Please simple comment. "Images"

File: practice-one/src/Components/Section/ProductList.jsx [hunk 7]
```diff
@@ -1,5 +1,7 @@
 import React from 'react';
-import dataCard from '../../../database/productCard.json';
+
+// Import Card data from the database
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/ProductList.jsx]: Please simple comment. "Mock database"

File: practice-one/src/database/footerIcon.js (hunk 4 above)
Comment (phattruong-agilityio) [reply practice-one/src/database/footerIcon.js:2]: Please simple comment.

File: practice-one/src/database/productCard.js (hunk 5 above)
Comment (phattruong-agilityio) [reply practice-one/src/database/productCard.js:2]: Please simple comment.

**Pull request merged**